packages/@core.keystone/setup.utils.js

```diff
@@ -56,9 +56,14 @@ function getAdapter (databaseUrl) {
 }
 
 function prepareDefaultKeystoneConfig (conf) {
```

Comment:
```
if (conf.PHASE === 'build') {
    console.warn('prepareDefaultKeystoneConfig() return nothing at build time')
    return {} // really we don't need any keystone settings at build time
}
```

Reply:
No, thats not working, keystone requires session store to be configured event at build time